### PR TITLE
[FIX] Accordion: Accepts just 1 child (#211)

### PR DESCRIPTION
**Pull request by vyshnavrajs2019**

### CODE REVIEW

**Comment by edwinbbu** [comment]
@goutham-subramanyam _a please review

File: lib/components/Accordion.js [hunk 1]
```diff
@@ -5,9 +5,17 @@ import Label from "./Label";
...
 const Accordion = ({ children, defaultActiveKey = null, className = "" }) => {
   const [openTab, setOpenTab] = useState(defaultActiveKey);
+  let childrenList = [];
+  if (children) {
+    if (children.__proto__.constructor.name === "Array") {
+      childrenList = children;
+    } else {
+      childrenList.push(children);
+    }
+  }
```
Comment (vinay0x) [inline lib/components/Accordion.js:15]: Check this out: https://stackoverflow.com/a/32705370 @vyshnavrajs2019 _a

**Comment by vinay0x** [comment]
@vyshnavrajs2019 _a 1) Mention Fixes #<pr number> in the title of the PR. 
2) @ mention someone and follow it with _a to assign it to them automatically. Check my previous comment.
3) Your Git credentials aren't properly set in your local. That's why your commits aren't properly mapped to your Github profile. Do:
```
git config --global user.email "youremail@example.com"
```
4) Look into the fix I mentioned in the review comment.

Once all this is done, please assign the PR to @goutham-subramanyam

**Comment by vyshnavrajs2019** [comment]
Fixes #211 
@goutham-subramanyam _a

**Review by goutham-subramanyam — APPROVED** [review]
LGTM.

**Comment by vinay0x** [comment]
@vyshnavrajs2019 Your commits are still not associated to your Github account. Please take care of this ASAP.

**Pull request merged**